packages/analytics-core/src/core-client.ts

```diff
@@ -34,15 +34,15 @@ export class AmplitudeCore<T extends Config> implements CoreClient<T> {
     this.name = name;
   }
 
-  _init(config: T) {
+  async _init(config: T) {
     this.config = config;
     this.timeline.reset();
     const queuedFunctions = this.q;
     this.q = [];
-    queuedFunctions.map(async (queuedFunction) => {
+    for (const queuedFunction of queuedFunctions) {
       await queuedFunction();
-    });
-    return Promise.resolve();
+    }
+    return undefined;
```

Comment:
This function will return `undefined` by default, so you might not need this line

Reply:
```suggestion
```